Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -98,7 +95,12 @@ namespace PhysX
             }
         }
 
-        
+        //! Function to add the ground for tests where the ground is needed
+        void AddGround(const AZ::Transform& floorTransform = DefaultFloorTransform)
```

Comment:
nit: It might be better to call this `SetGround` as calling this multiple times will override the same value (and not append multiple floors)

Reply:
Renamed to SetupGround and added better comments.